packages/opentelemetry-exporter-prometheus/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@opentelemetry/exporter-prometheus",
+  "version": "0.0.1",
+  "description": "OpenTelemetry Exporter Prometheus provides a metrics endpoint for Prometheus",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "clean": "rimraf build/*",
+    "check": "gts check",
+    "compile": "tsc -p .",
+    "fix": "gts fix"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "tracing",
```

Comment:
`s/tracing/metrics`?

Reply:
There was both in there but I removed tracing now - we can add it again anytime.